### ISSUE TITLE
Why am I not able to achieve the same results when using the CORD training code provided by you?

### ISSUE THREAD

**Issue by Masterchenyong** [clovaai/donut#210]
python train.py --config config/train_cord.yaml \
        --pretrained_model_name_or_path "/home/kas/chenyong/HuggingFaceModels/others/donut-base" \
        --dataset_name_or_paths '["naver-clova-ix/cord-v2"]' \
        --exp_version "test_experiment" 
  
The minimum loss value ：10.5

![image](https://github.com/clovaai/donut/assets/33871334/82ab6e75-3e57-474d-9895-447c9ede7440)

**Comment by dscrlc** [comment]
have you solved this problem? i have same problem,logs here:
    Answer: <s_menu><s_nm>TT</s_nm><s_unitprice>20,000</s_unitprice><s_cnt>1</s_cnt><s_price>20,000</s_price></s_menu><s_total><s_total_price>20,000</s_total_price><s_cashprice>100,000</s_cashprice><s_changeprice>80,000</s_changeprice></s_total>
 Normed ED: 0.9756309834638817
Prediction: <s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s><s>
    Answer: <s_menu><s_nm>PKT TELOR/PERKEDEL</s_nm><s_price>26,000</s_price><sep/><s_nm>TERONG</s_nm><s_price>12,000</s_price><sep/><s_nm>PARU</s_nm><s_price>23,000</s_price><sep/><s_nm>SBL GR ATI/AMPLA</s_nm><s_price>20,000</s_price><sep/><s_nm>NESTLE 330 ML</s_nm><s_price>8,000</s_price></s_menu><s_sub_total><s_subtotal_price>89,000</s_subtotal_price><s_tax_price>8,900</s_tax_price></s_sub_total><s_total><s_total_price>97,900</s_total_price><s_cashprice>100,000</s_cashprice><s_changeprice>2,100</s_changeprice><s_menuqty_cnt>5.00</s_menuqty_cnt></s_total>
 Normed ED: 0.9425587467362925

loss=11

with "windows10"/"distributed_backend=gloo"  /"train_batch_sizes=4".  I don't know where the problem lies, why the loss value is so high
